client/src/components/dashboard/flights/addFlight.js

```diff
@@ -223,26 +231,11 @@ const AddFlight = ({
             margin="dense"
             id="flightNumber"
             name="flightNumber"
-            label="Flight Number"
+            label="Look Up Flight"
```

Comment:
Can we please have the label for this field remain as 'Flight Number', and perhaps include a text field above this section to tell users they can enter flight deets to search for their flights?

Reply:
This is my fault, I misunderstood Dennis when he asked me earlier! Apologies Dennis but yes Kathleen's request makes sense.